### ISSUE TITLE
Tutorials broken since move to new build process

### ISSUE THREAD

**Issue by bwbohl** [music-encoding/music-encoding.github.io#393]
apparently the build flattens the folder structure of the tutorials and thus renders all relative URIs to the respective resources invalid.

**Comment by rettinghaus** [comment]
Jekyll reports a conflict:

```shell
Generating... 
[101](https://github.com/rettinghaus/music-encoding.github.io/actions/runs/3242797360/jobs/5316595112#step:5:102)
          Conflict: The following destination is shared by multiple files.
[102](https://github.com/rettinghaus/music-encoding.github.io/actions/runs/3242797360/jobs/5316595112#step:5:103)
                    The written file may end up with unexpected contents.
[103](https://github.com/rettinghaus/music-encoding.github.io/actions/runs/3242797360/jobs/5316595112#step:5:104)
                    /github/jekyll_build/tutorials/tutorials.xml
[104](https://github.com/rettinghaus/music-encoding.github.io/actions/runs/3242797360/jobs/5316595112#step:5:105)
                     - /github/workspace/_tutorials/199_tutorials/tutorials.xml
[105](https://github.com/rettinghaus/music-encoding.github.io/actions/runs/3242797360/jobs/5316595112#step:5:106)
                     - /github/workspace/_tutorials/180_understanding-odd/tutorials.xml
[106](https://github.com/rettinghaus/music-encoding.github.io/actions/runs/3242797360/jobs/5316595112#step:5:107)
                    
[107](https://github.com/rettinghaus/music-encoding.github.io/actions/runs/3242797360/jobs/5316595112#step:5:108)
          Conflict: The following destination is shared by multiple files.
[108](https://github.com/rettinghaus/music-encoding.github.io/actions/runs/3242797360/jobs/5316595112#step:5:109)
                    The written file may end up with unexpected contents.
[109](https://github.com/rettinghaus/music-encoding.github.io/actions/runs/3242797360/jobs/5316595112#step:5:110)
                    /github/jekyll_build/tutorials/just-a-name.xml
[110](https://github.com/rettinghaus/music-encoding.github.io/actions/runs/3242797360/jobs/5316595112#step:5:111)
                     - /github/workspace/_tutorials/199_tutorials/just-a-name.xml
[111](https://github.com/rettinghaus/music-encoding.github.io/actions/runs/3242797360/jobs/5316595112#step:5:112)
                     - /github/workspace/_tutorials/180_understanding-odd/just-a-name.xml
[112](https://github.com/rettinghaus/music-encoding.github.io/actions/runs/3242797360/jobs/5316595112#step:5:113)
                    
[113](https://github.com/rettinghaus/music-encoding.github.io/actions/runs/3242797360/jobs/5316595112#step:5:114)
                    done in 2.599 seconds.
[114](https://github.com/rettinghaus/music-encoding.github.io/actions/runs/3242797360/jobs/5316595112#step:5:115)
```

**Comment by bwbohl** [comment]
closed with #394
